README.md

```diff
@@ -1,14 +1,5 @@
-# lambda-template
-Template for creating lambda repositories
-A template for quickly starting a new AWS lambda project.
-
-## Naming
-Naming conventions:
-* for a vanilla Lambda: `lambda-<context>`
-* for a Cloudformation Transform macro: `cfn-macro-<context>`
-* for a Cloudformation Custom Resource: `cfn-cr-<context>`
-
-## Development
+# lambda-send-budget-alert
+A Lambda that listens to an SNS topic and forwards notifications to Synapse users by email
```

Comment:
This says what the lambda does but could you also please describe why we even need this lambda?  What is the problem that we are solving with lambda?

Reply:
Done!  See the latest commit, https://github.com/Sage-Bionetworks-IT/lambda-send-budget-alert/pull/1/commits/4b1ca9b6f5c942a09dec09c5360d7417f04b8955